docs/tutorials/license-server-auth.md

```diff
@@ -89,6 +89,10 @@ We can use a request filter to modify the URL and add the required parameter:
     // Only add headers to license requests:
     if (type == shaka.net.NetworkingEngine.RequestType.LICENSE) {
       // This is the specific parameter name and value the server wants:
+      // Note that there would be multiple uris for license server.
+      //   e.g. one for Widevine and one for PlayReady
```

Comment:
I think this part is confusing.  It implies that uris[0] might be Widevine, for example, while uris[1] might be PlayReady.  This is not the case.  For a license request, uris.length will always be 1, and uris[0] will always be the license server URI for the chosen key system.